.eslintrc.js

```diff
@@ -376,5 +375,16 @@ module.exports = {
                 'import/no-extraneous-dependencies': 0,
             },
         },
+        {
+            files: [
+                'rollup.config.mjs',
+                'packages/core/**/*',
+                'packages/webpack-plugin/**/*',
+                'packages/esbuild-plugin/**/*',
+            ],
+            rules: {
+                'import/no-extraneous-dependencies': 0,
```

Comment:
```suggestion
                'import/no-extraneous-dependencies': 'error',
```

Reply:
Fixed in https://github.com/DataDog/build-plugin/pull/67
In this case, you wanted it to be `'off'` right?